src/tree/registries/Azure/AzureRegistryDataProvider.ts

```diff
@@ -63,7 +67,8 @@ export class AzureRegistryDataProvider extends RegistryV2DataProvider implements
                     label: sub.name,
                     type: 'azuresubscription',
                     subscription: sub,
-                    additionalContextValues: ['azuresubscription']
+                    additionalContextValues: ['azuresubscription'],
+                    iconPath: vscode.Uri.joinPath(this.extensionContext.extensionUri, 'resources', 'azureSubscription.svg'),

```

Comment:
this is where the icon will be used (subscription tree item)

Reply:
look something like this
![image](https://github.com/microsoft/vscode-docker/assets/59073590/0bd2202a-812e-4bbf-9005-fbfae8d663f5)